### PR TITLE
Add link to display all the possible features

### LINKED ISSUE #2087: [query tool] get rid of maximum number of results by introducing a "load more results" button
UC: open app, load layer ch.bazl.luftfahrthindernis, search for all objects with AGL altitude > 100m
Result: the first 200 results are listed and highlighted. In reality there are more results on that query and the user has no chance to access the further results
Expected: first 200 results are listed and highlighted and the user can load the previous/next 200 results with a paging (like in old mf-bazl)

To be discussed:
But according to the feedbacks this will be the main improvement bazl will want in the meeting next week
IMHO this is valuable for all other layers which will be queried
Do you see any technical difficulties?

Thanks for your 2 cents

### PR DESCRIPTION
**Pull request by oterral**
Fix #2087 , #2094 

[Test](http://mf-geoadmin3.dev.bgdi.ch/teo_results/?X=190000.00&Y=660000.00&zoom=1&lang=fr&topic=aviation&bgLayer=ch.swisstopo.pixelkarte-grau&catalogNodes=1379,1381,1514&layers=ch.bazl.luftfahrthindernis)

Work with https://github.com/geoadmin/mf-chsdi3/pull/1266

### CODE REVIEW

**Comment by oterral** [comment]
Branch updated with the more results button.

If the branch link doesn't work test on my home directory http://mf-geoadmin3.dev.bgdi.ch/ltteo/

**Comment by oterral** [comment]
I think it's good now. @mariokeusen @davidoesch @cedricmoullet ? what do you think ?

**Comment by davidoesch** [comment]
it is now a paging -- user can *view* more than 50 results. The issue comes
when I hit printing ( print icon in the query result) here more than 50 are
shown...and printable.as soon as we have the csv export: we should only
allow 50 for export and the use a real paging.
If someone wants to download all the data: use data.geo.admin.ch

**Comment by mariokeusen** [comment]
Agree with David's comment. 
In addition what is important for me: the user should get the total number of results displayed on top of the window in a way it gets obvious that his query produces more than 200 results. 
E.g. in the top text: "objets (200/279)

**Comment by gjn** [comment]
To know the total results is a performance issue. I suggest that we do some performance tests and decide upon them if we

1) show "objects 200/total"
2) show "objects 200, show more"

2) is certainly faster, potentially much faster.

**Comment by mariokeusen** [comment]
Good for me, let's test and decide then.

**Comment by davidoesch** [comment]
+1

**Comment by gjn** [comment]
I think the indication that we have more than 200 results should be in a) objects tab header and b) layer entry (maybe bold text, or asterisk next to 200?)

It's cumbersome to scroll to the button to see if there are more results.

File: src/components/query/QueryDirective.js [hunk 1]
```diff
@@ -210,71 +186,100 @@
...
+    };
+
     // Search by geometry using feature identify servioce
-    $scope.searchByGeometry = function() {
+    $scope.searchByGeometry = function(layerBodId, offset) {
       $scope.queryType = 0;
 
       if ($scope.selectByRectangleLayers.length == 0) {
-        $scope.options.features = [];
+        onSearchError();
```
Comment (gjn) [inline src/components/query/QueryDirective.js]: Not really an error here, is it? maybe call the function resetResults()?

File: src/components/query/QueryDirective.js [hunk 2]
```diff
@@ -210,71 +186,100 @@
...
       $scope.queryType = 1;
       $scope.loading = true;
 
-      var features = [];
       var params = getParamsByLayer($scope.filters);
       if (params.length == 0) {
         if ($scope.useBbox) {
           $scope.searchByGeometry();
+        } else {
+          onSearchError();
```
Comment (gjn) [inline src/components/query/QueryDirective.js]: Same as above. Not really an error.

**Comment by gjn** [comment]
Usability comments and a few minor code comments. Otherwise, code looks very well.

I would really update the UI to better indicate that there are more results.

**Comment by davidoesch** [comment]
+1 for Gilbert UI comments..(a) objects tab header and b) layer entry (maybe bold text, or asterisk next to 200)

Another proposal : if more than xx results available : the first line in the results list contains the text " first  xxx results of zzz shown, scroll down for more @

**Comment by gjn** [comment]
As mentioend in previous comment, the 'of zzz' has performance implications...

**Comment by davidoesch** [comment]
"only first  xxx results shown, scroll down for more "

File: src/components/query/QueryDirective.js (hunk 1 above)
Comment (oterral) [inline src/components/query/QueryDirective.js]: Yes better name

**Comment by oterral** [comment]
Can we just add a`+` before the 200 (accordion header and layer header) ?

**Comment by gjn** [comment]
yes, I think that would be enough.

**Comment by oterral** [comment]
PR upadted with the `+` stuff

**Comment by gjn** [comment]
That's good. Please squash and merge.

**Comment by oterral** [comment]
thx for the review

**Comment by davidoesch** [comment]
nice!

**Pull request merged**